e2e_tests_terraform.cloudbuild.yaml

```diff
@@ -1,8 +1,41 @@
 steps:
     # remove any detritus left from prior runs
   - id: Clean
-    name: 'gcr.io/cloud-builders/gcloud'
-    args: ['builds','submit','--config=e2e_tests_tf_cleanup.cloudbuild.yaml','--substitutions=_TARGET_PROJECT=$_TARGET_PROJECT']
+    name: gcr.io/$PROJECT_ID/fourkeys-builder
```

Comment:
Any reason why we got rid of the cleanup file?

Reply:
I hadn't found any need to run the cleanup build config on its own, so I inlined it into the primary config. Happy to keep it separate, if you foresee reasons to run the cleanup on its own.